spec/rspec/rails/example/controller_example_group_spec.rb

```diff
@@ -21,9 +21,8 @@ def group_for(klass)
 
     it "handles methods invoked via `method_missing` that use keywords" do
       example = group.new
-      example.define_singleton_method(:has_val?) { |val:| val == 1 }
-      expect(example).to example.have_val(val: 1)
-      expect(example).to_not example.have_val(val: 2)
+      example.define_singleton_method(:keyword_value) { |value:| value }
+      expect(example.keyword_value(value: 42)).to eq 42
```

Comment:
I'll roll this change back, it fails to protect from regression. The reason is that `method_missing` is not called, because we call the same method that we define. Previously, we were defining `has_val?` and were calling `have_val`. `super` then calls a correct method thanks to dynamic matchers.